components/license-control-service/tools/gen-keys/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/sirupsen/logrus"
```

Comment:
💭 Does such a small utility need logrus? Fine. 😉

Reply:
Not directly related to the PR, but...do we *have* a standard for what logging framework to use when?

Reply:
I don't think so. But also, I guess I shouldn't scold anyone for using what we use everywhere everywhere 😉 :ignoreme: